src/Essentials/samples/Samples/Startup.cs

```diff
@@ -32,10 +33,24 @@ public static MauiApp CreateMauiApp()
 						.AddWindows(windows => windows
 							.OnLaunched((app, e) =>
 								Microsoft.Maui.Essentials.Platform.OnLaunched(e)));
+#elif TIZEN
+					Microsoft.Maui.Essentials.Platform.Init(CoreUIAppContext.GetInstance(MauiApplication.Current).MainWindow);
 #endif
 				})
 				.UseMauiApp<App>();
 
+#if TIZEN
+			appBuilder
+				.Services.AddTransient((_) =>
+				{
+					var option = new InitializationOptions
```

Comment:
Is this correct? Is this used outside of `Forms.Init()`?

Reply:
Yes. This is intended.

As you know, app developer doesn't have to use `Forms.Init()` explicitly, because `Forms.Init()` is being called internally through Compatibility's AppHostBuilderExtensions logic. 
https://github.com/rookiejava/maui/blob/ab3b9e7a8d7507883e6f1b547883b7daf790f05f/src/Compatibility/Core/src/AppHostBuilderExtensions.Tizen.cs#L26-L40

If you want to Opt-in Init, you just to add InitiaizationOption to the services as shown above.  
Because this is sample app, we just want too show how we can opt-in init. In most cases, of course, app works well without this code.